src/content/docs/style-guide/structure/styleguide-markup-indentation.mdx

```diff
@@ -0,0 +1,364 @@
+---
+title: "Markdown indentation"
+tags:
+  - Basic style guide
+  - Style guide quick reference
+---
+
+import indentSnippetsInLists from 'images/indent-snippets-in-lists.png'
+
+import indentLeftJustified from 'images/indent-left-justified.png'
+
+import indentExtraLineFeed from 'images/indent-extra-line-feed.png'
+
+import topology1 from 'images/topology-1.png'
+
+import topology2 from 'images/topology-2.png'
+
+import childBulletList from 'images/child-bullet-list.png'
+
+import childNumberedList from 'images/child-numbered-list.png'
+
+import fourSpaceIndentChild from 'images/four-space-indent-child.png'
+
+import threeSpaceIndentChild from 'images/three-space-indent-child.png'
+
+import basicListIndentationLeft from 'images/basic-list-indentation-left.png'
+
+import basicListIndentationTwo from 'images/basic-list-indentation-two.png'
+
+
+When you're writing in Markdown, keep in mind that indentation can make your mark-up more readable, and it can also affect its readability when it's rendered in HTML.
+
+## First-level lists (parents) [#first-level]
+
+For first-level numbered and bulleted lists introduced by a left-justified introductory sentence, we follow the common Markdown standard: left-justify the first-level list items. At publication, the Markdown library will indent the first-level list items for you:
+
+```
+Here is a basic list written as left-justified but indented at the time of publication:
+* Item 1
+* Item 2
+```
+
+This is rendered like this:
+
+<img
+  title="Screenshot showing poor formatting of child bullet list"
+  alt="Screenshot showing poor formatting of child bullet list"
+  src={basicListIndentationLeft}
+/>
+
+<Callout variant="tip">
```

Comment:
Our style guide mentions only including 1 call-out per doc. I think the information in this callout could be removed, or clarified. What does "less semantically accurate" mean? If it's more of a nitpicky detail situation, I'd remove this call out. If it can ruin the structure of a list, then I'd specify how.

Reply:
@akristen OK, I removed this since it was distracting!